src/spring-cloud/azext_spring_cloud/application_configuration_service.py

```diff
@@ -0,0 +1,224 @@
+from .vendored_sdks.appplatform.v2022_05_01_preview import models as models
+from azure.cli.core.commands import cached_put
+from azure.cli.core.util import sdk_no_wait
+from knack.log import get_logger
+from knack.util import CLIError
+
+APPLICATION_CONFIGURATION_SERVICE_NAME = "ApplicationConfigurationService"
+
+logger = get_logger(__name__)
+
+def application_configuration_service_show(cmd, client, service, resource_group):
+    if is_enterprise_tier(cmd, resource_group, service):
+        return client.configuration_services.get(resource_group, service)
+    else:
+        raise CLIError("Application Configuration Service is only supported in Enterprise tier.")
+
+
+def application_configuration_service_clear(cmd, client, service, resource_group):
+    if is_enterprise_tier(cmd, resource_group, service):
+        properties = models.ConfigurationServiceGitProperty()
+        acs_resource = models.ConfigurationServiceResource(properties=properties)
+        return client.configuration_services.begin_create_or_update(resource_group, service, acs_resource)
+    else:
+        raise CLIError("Application Configuration Service is only supported in Enterprise tier.")
+
+
+def application_configuration_service_git_add(cmd, client, service, resource_group,
+                                              name, patterns, uri, label,
+                                              search_paths=None,
+                                              username=None,
+                                              password=None,
+                                              host_key=None,
+                                              host_key_algorithm=None,
+                                              private_key=None,
+                                              strict_host_key_checking=None,
+                                              no_wait=False):
+    if patterns:
+        patterns = patterns.split(",")
+    repo = models.ConfigurationServiceGitRepository(name=name, patterns=patterns, uri=uri, label=label)
+
+    if search_paths:
+        search_paths = search_paths.split(",")
+    repo.search_paths = search_paths
+    repo.username = username
+    repo.password = password
+    repo.host_key = host_key
+    repo.host_key_algorithm = host_key_algorithm
+    repo.private_key = private_key
+    repo.strict_host_key_checking = strict_host_key_checking
+
+    acs_resource = client.configuration_services.get(resource_group, service)
+    if acs_resource is None or acs_resource.properties is None:
+        raise CLIError("Application Configuration Service is not enabled.")
+    
+    acs_settings = acs_resource.properties.settings
+
+    if acs_settings is None or acs_settings.git_property is None or acs_settings.git_property.repositories is None:
+        repos = []
+        git_property = models.ConfigurationServiceGitProperty(repositories=repos)
+        acs_settings = models.ConfigurationServiceSettings(git_property=git_property)

```

Comment:
will this covers existing or potential existing settings except for `git_properties`?

Reply:
Currently only `git_property `supported in `acs_settings`, so won't impact existing property.

Reply:
what if a new property added?